styles.css

```
@@ -0,0 +1,61 @@
+@import url(https://fonts.google.com/specimen/Oswald);
+
+
+body {
+    font-family: "Oswald", sans-serif;
+    background:#ecf0f1; 
+    padding: 20px;
+    text-align: center;
+    justify-content: center;
```

Para usar justify-content: center debe utilizar display: flex;

```suggestion
    display: flex;
    justify-content: center;
```